libraries/lib-wave-track/WideClip.cpp

```diff
@@ -56,3 +56,17 @@ double WideClip::GetStretchRatio() const
 {
    return mChannels[0u]->GetStretchRatio();
 }
+
+double WideClip::GetSemitoneShift() const
+{
+   return mChannels[0u]->GetSemitoneShift();
+}
+
+void WideClip::SetPitchShiftChangePublisher(
+   std::weak_ptr<PitchShiftChangePublisher> publisher)
+{
+   // On purpose set the publisher on the left channel only. This is not a clip
+   // property that is saved to disk, and else we'll get two callbacks for the
+   // same event.
```

Comment:
What happens if you swap channels?

Reply:
This method is called when preparing tracks for playback. During playback, swapping channels isn't possible. After playback, the publisher will have expired, so swapping channels wouldn't have consequence.